### PR TITLE
Content blocks export

### PR DESCRIPTION
**Pull request by aapomm**
### How to test
1. Open a project and create content blocks in it.
2. Export the project using template and package.
3. Assert that the generated XML contains the content block data.

### CODE REVIEW

File: lib/dradis/plugins/projects/export/v1/template.rb [hunk 1]
```diff
@@ -122,6 +122,22 @@ def build_notes_for_node(builder, node)
...
+    def build_content_blocks(builder)
+      return unless defined?(ContentBlock)
+
+      content_blocks = ContentBlock.all
+      builder.content_blocks do |blocks_builder|
+        content_blocks.each do |block|
+          blocks_builder.content_block do |block_builder|
+            block_builder.id(block.id)
+            block_builder.author(block.author)
+            block_builder.kind(block.kind)
```
Comment (dormi) [inline lib/dradis/plugins/projects/export/v1/template.rb]: `kind` has been renamed to `name`

File: lib/dradis/plugins/projects/export/v1/template.rb [hunk 2]
```diff
@@ -122,6 +122,22 @@ def build_notes_for_node(builder, node)
...
+            block_builder.text(block.text)
```
Comment (dormi) [inline lib/dradis/plugins/projects/export/v1/template.rb]: `text` has been renamed to `content`

Comment (dormi) [reply lib/dradis/plugins/projects/export/v1/template.rb]: Also, `author` returns a `User` object

File: lib/dradis/plugins/projects/export/v1/template.rb [hunk 3]
```diff
@@ -122,6 +122,22 @@ def build_notes_for_node(builder, node)
...
+      return unless @project
+      content_blocks = @project.content_blocks
+
+      builder.content_blocks do |blocks_builder|
+        content_blocks.each do |block|
+          blocks_builder.content_block do |block_builder|
+            block_builder.id(block.id)
+            block_builder.author(block.author.try(:email))
+            block_builder.name(block.name)
+            block_builder.content(block.content)
```
Comment (dormi) [inline lib/dradis/plugins/projects/export/v1/template.rb]: Just noticed that we should export this as a CDATA field

Comment (dormi) [reply lib/dradis/plugins/projects/export/v1/template.rb]: also, the version should be `3.8.0` ?

File: lib/dradis/plugins/projects/export/template.rb [hunk 4]
```diff
@@ -20,6 +21,9 @@ def build_categories(builder);    raise NotImplementedError; end
     def build_issues(builder);        raise NotImplementedError; end
     def build_methodologies(builder); raise NotImplementedError; end
     def build_nodes(builder);         raise NotImplementedError; end
+    def build_content_blocks(builder)
+      raise NotImplementedError if version > 1
```
Comment (etdsoft) [inline lib/dradis/plugins/projects/export/template.rb]: I'd no-op this in v==1 instead of doing this weird check here.

**Comment by etdsoft** [comment]
Sorry for this @dormi , but we're not really exporting ContentBlock data here, we're aiming to export "Report content" right?

If that's the case, I'd propose we rename the methods throughout to make it more future-proof / all encompassing.

After that, this PR gets my 👍 for merge.

**Pull request merged**